bindings_ffi/src/mls.rs

```diff
@@ -1157,26 +1157,10 @@ impl From<&FfiMetadataField> for MetadataField {
 
 #[uniffi::export(async_runtime = "tokio")]
 impl FfiConversations {
-    pub async fn create_group(
+    pub async fn create_group_optimistic(
```

Comment:
Does this need to be `async`? That may give integrators the wrong impression

Reply:
good catch, will fix!